qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/model.py

```diff
@@ -630,20 +640,21 @@ def move_output_files_to_user_save_location(self):
     def load_basic_outputs(self, gsas_result_filepath):
         logger.notice(f"GSAS-II .lst result file found. Opening {self.project_name}.lst")
         self.read_gsas_lst_and_print_wR(gsas_result_filepath, self.data_files)
+        self.phase_names_list = self.find_phase_names_in_lst(
+            os.path.join(self.user_save_directory, self.project_name + ".lst"))
```

Comment:
Should this be the temporary directory?

Reply:
Moved it until after the files are in the final save directory instead!